### PR TITLE
[NEW] Record audio & send it

### LINKED ISSUE #1312: [NEW] Record an audio & send it

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios 

TODO:
- [x] Make it work with themes
- [x] Point to master RocketChatViewController branch

Closes #1312 

To be reviewed & merged along with https://github.com/RocketChat/RocketChatViewController/pull/51

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ def shared_pods
   pod 'semver'
 
   # UI
-  pod 'RocketChatViewController', :git => 'https://github.com/RocketChat/RocketChatViewController'
+  pod 'RocketChatViewController', :path => '~/dev/RocketChat/RocketChatViewController' # :git => 'https://github.com/RocketChat/RocketChatViewController'
```
Comment (rafaelks) [inline Podfile]: 👀

**Comment by steve7688** [comment]
How to take an audio and send it?Where is the entrance?

**Comment by cardoso** [comment]
> How to record an audio and send it?Where is the entrance?
> 
> I run the code from branch feat/record_audio.1312

Did you point the Podfile to this branch? https://github.com/RocketChat/RocketChatViewController/pull/51

The entry-point should be the mic icon when text field is empty.

**Comment by steve7688** [comment]
@cardoso YES

pod 'RocketChatViewController', :git => 'https://github.com/RocketChat/RocketChatViewController'

What should i do next?

Thanks!

**Comment by cardoso** [comment]
@steve7688 I think you're forgetting to point to the correct branch

pod 'RocketChatViewController', :git => 'https://github.com/RocketChat/RocketChatViewController', :branch => 'feat/audio_record'

**Comment by cardoso** [comment]
@steve7688 it will be soon ;) just need to decide on a few details and make this work with themes.

**Pull request merged**